package.json

```diff
@@ -1,22 +1,24 @@
 {
-  "name": "country-identity-packages",
+  "name": "anon-aadhaar",
   "version": "0.1.15",
   "description": "A set of tools to anonymously prove your identity",
   "main": "index.js",
   "private": true,
+  "license": "MIT",
   "scripts": {
     "build:libraries": "yarn workspaces run build",
     "test-publish:libraries": "yarn workspaces run test-publish",
-    "publish:libraries": "yarn workspace anon-aadhaar-pcd run publishtonpm && yarn workspace anon-aadhaar-react run publishtonpm && yarn workspace anon-aadhaar-contracts run publishtonpm",
+    "publish:libraries": "yarn workspaces run publishtonpm",
     "test:libraries": "yarn workspaces run test",
-    "postversion": "yarn workspace anon-aadhaar-pcd version --new-version $npm_package_version --no-git-tag-version && yarn workspace anon-aadhaar-react version --new-version $npm_package_version --no-git-tag-version && yarn workspace anon-aadhaar-contracts version --new-version $npm_package_version --no-git-tag-version",
+    "bump": "yarn version --no-commit-hooks --no-git-tag-version",
+    "version": "yarn workspaces run bump-to $npm_package_version",
     "lint": "yarn workspaces run lint",
     "prepare": "husky install",
-    "dev-install:circuit": "yarn workspace anon-aadhaar-circuit run dev-install",
-    "dev-setup:circuit": "yarn workspace anon-aadhaar-circuit run dev-setup",
-    "witness:circuit": "yarn workspace anon-aadhaar-circuit run gen-witness",
-    "proof:circuit": "yarn workspace anon-aadhaar-circuit run gen-proof",
-    "verif:circuit": "yarn workspace anon-aadhaar-circuit run verif-proof"
+    "dev-install:circuit": "yarn workspace @anon-aadhaar/circuits run dev-install",
+    "dev-setup:circuit": "yarn workspace @anon-aadhaar/circuits run dev-setup",
+    "witness:circuit": "yarn workspace @anon-aadhaar/circuits run gen-witness",
+    "proof:circuit": "yarn workspace @anon-aadhaar/circuits run gen-proof",
+    "verif:circuit": "yarn workspace @anon-aadhaar/circuits run verif-proof"
```

Comment:
Not related to this PR, but do we need these things on the root package.json. Maybe keep it only for scripts that applied to the all packages.

Reply:
They are here for the CI/CD scripts and also to onboard someone who would want to start working on the repo.